tools/helios-build/src/main.rs

```diff
@@ -1956,92 +1918,154 @@ fn cmd_image(ca: &CommandArg) -> Result<()> {
+        } else {
+            top_path(&["image", "amd", &efs])?
+        };
+        if !efsfile.is_file() {
+            bail!("AMD configuration file {efs:?} does not exist");
+        }
+        info!(log, "  efs: {efsfile:?}");
+        let fws = board_obj
+            .get("firmware")
+            .and_then(|v| v.as_array())
+            .ok_or_else(|| {
+                anyhow::anyhow!("Missing or invalid 'firmware' array for board")
+            })?;
+
+        let amdblobs = fws
+            .iter()
+            .map(|y| {
+                let y = y.as_str().ok_or_else(|| {
+                    anyhow::anyhow!("Invalid firmware entry: {:?}", y)
+                })?;
+                /*
+                 * If the entry is an absolute path, just pass it
+                 * through. Otherwise look in the place where we
+                 * usually store the firmware blobs:
+                 */
+                let d = if y.starts_with("/") {
+                    let p = PathBuf::from(y);
+                    assert!(p.is_absolute());
+                    p
+                } else {
+                    top_path(&["projects", "amd-firmware", &y])?
+                };
+                info!(log, "  firmware: {d:?}");
+                Ok(d)
+            })
+            .collect::<Result<Vec<_>>>()?;
+        let missing =
+            amdblobs.iter().filter(|d| !d.is_dir()).collect::<Vec<_>>();
+        if !missing.is_empty() {
+            bail!(
+                "These AMD firmware blob directories do not exist {missing:?}"
+            );
+        }
 
+        info!(log, "building host image for board {board}...");
+        let romname = &format!("rom.{board}");
+        let rom = rel_path(Some(&outdir), &[romname])?;
+        let reset = top_path(&[
+            "projects",
+            "phbl",
+            "target",
+            "x86_64-oxide-none-elf",
+            "release",
+            "phbl",
+        ])?;
+        let ahibdir = top_path(&["projects", "amd-host-image-builder"])?;
+        let ahibargs_base = {
+            let mut t: Vec<String> = vec![ahib.as_str().into()];
+            for blob in amdblobs {
+                t.push("--blobdir".into());
+                t.push(blob.to_str().unwrap().into());
+            }
+            t
+        };
+        let ahibargs = {
+            let mut t = ahibargs_base.clone();
+            t.push("--config".into());
+            t.push(efsfile.to_str().unwrap().into());
+            t.push("--output-file".into());
+            t.push(rom.to_str().unwrap().into());
+            t.push("--reset-image".into());
+            t.push(reset.to_str().unwrap().into());
+            t
+        };
+        ensure::run_in(
+            log,
+            &ahibdir,
+            &ahibargs.iter().map(String::as_str).collect::<Vec<_>>(),
+        )?;
+        tar.add_file(&rom, romname)?;
+        // For now, add the gimlet rom again as just "rom".
+        // Tufaceaous doesn't parse symlinks.
```

Comment:
```suggestion
        /*
         * Add the Gimlet ROM image again at the original path "rom" for
         * compatibility with older tools that do not understand multi-image
         * archives:
         */
```

I suspect tufaceaous is probably not the only tool that would be confused by this being a symlink entry instead of a file, FWIW.